huxley/api/views/delegate.py

```diff
@@ -35,4 +35,4 @@ class DelegateDetail(generics.RetrieveUpdateDestroyAPIView):
     authentication_classes = (SessionAuthentication, )
     queryset = Delegate.objects.all()
     permission_classes = (permissions.DelegateDetailPermission, )
-    serializer_class = DelegateSerializer
+    serializer_class = DelegateSerializer
```

Comment:
revert, and/or configure your editor to automatically add a newline at the end of files.